docs/constants-and-vars.rst

```diff
@@ -25,6 +25,7 @@ Name                 Type             Value
 ``msg.sender``       ``address``      Sender of the message (current call)
 ``msg.value``        ``uint256``      Number of wei sent with the message
 ``tx.origin``        ``address``      Sender of the transaction (full call chain)
+``tx.gasprice``      ``uint256``      Gas price of current transaction in wei
```

Comment:
What is this set to post-EIP1559?

Reply:
For 1559 (type 2) txs, it'll just match `BASEFEE`, but for legacy txs it'll be whatever gasprice was set when the tx was built.

https://eips.ethereum.org/EIPS/eip-1559#gasprice

Reply:
Mainly useful for EVM chains that don't support LONDON yet

Reply:
perfect